.tekton/collector-component-pipeline.yaml

```diff
@@ -610,7 +613,7 @@ spec:
       - name: name
         value: push-dockerfile-oci-ta
       - name: bundle
-        value: quay.io/konflux-ci/tekton-catalog/task-push-dockerfile-oci-ta:0.1@sha256:98ccae6ac132ab837fc51a70514be5fca656e09d6d4ad93230bd10f0119258aa
+        value: quay.io/konflux-ci/tekton-catalog/task-push-dockerfile-oci-ta:0.1@sha256:a2beb43c9f2a72f55ca17e196f66bcdaf4ff9a0b722c7e063af1f38e7003faad
```

Comment:
Does this need to be updated?

Reply:
Actually I had to bump it to the most recent at that time because builds were failing with strange errors.
See [`735e99d` (#2001)](https://github.com/stackrox/collector/pull/2001/commits/735e99da3086a4a2ee39445cff52bca3209fbf1a)

There's currently even newer digest but Renovate can also take care about it.